test-docs/r-quickstart.R

```diff
@@ -1,5 +1,8 @@
 library(laminr)
+laminr::lamin_init(storage = "./test-docs-2", modules = c("bionty"))
```

Comment:
Obviously this shouldn't be necessary but it seems that environments cannot necessarily be re-used across steps in GitHub Action.